src/git/zif_abapgit_git_definitions.intf.abap

```diff
@@ -92,7 +92,9 @@ INTERFACE zif_abapgit_git_definitions
       chmod TYPE ty_chmod,
     END OF ty_expanded .
   TYPES:
-    ty_expanded_tt TYPE STANDARD TABLE OF ty_expanded WITH DEFAULT KEY .
+    ty_expanded_tt TYPE STANDARD TABLE OF ty_expanded WITH DEFAULT KEY
+      WITH NON-UNIQUE SORTED KEY path_name COMPONENTS path name
+      WITH NON-UNIQUE SORTED KEY path COMPONENTS path.
```

Comment:
now the "path" secondary sorting can be removed? as its already the first part in "path_name"?

Reply:
There's a loop with only a condition on "path" (no "name"). it gives syntax error if you try to use the "path_name" key. 

I don't have a big repo to test the impact. I will ask the reporter to test it

Reply:
hmm, okay